src/main/java/org/broadinstitute/hellbender/tools/variantdb/nextgen/ExtractFeatures.java

```diff
@@ -65,6 +65,12 @@
         optional = true)
     protected double hqGenotypeABThreshold = 0.2;
 
+    @Argument(
+        fullName = "excess-alleles-threshold",
+        doc = "excess alleles threshold", // TODO what even is this?!??!
```

Comment:
something like "number of non-reference alleles above which a site will be filtered out." ?

also @kcibul should we add an option to NOT filter these at all? maybe by setting this to -1 or something?

Reply:
I'd say that's a nice to have... by having the threshold we are more configurable than before AND we can effectively disable it by setting this to a very large number.